src/ports/postgres/modules/deep_learning/keras_model_arch_table.sql_in

```diff
@@ -333,7 +333,7 @@ SELECT COUNT(*) FROM model_arch_library;
 <pre class="result">
  count
 -------+
-     1
+     2
```

Comment:
This should be 1 since we delete a model out of 2

Reply:
we delete 1 model out of 3 actually, so 2 is correct.

the query just above in step 3 is 
```
SELECT COUNT(*) FROM model_arch_library WHERE model_weights IS NOT NULL;
```
which just counts models that have non-null wts (there are 2), not the total list of models (there are 3 at that point)

Reply:
Oh I see, thanks for pointing it out.